### PR TITLE
Add debug command to get filesystem metadata

### PR DESCRIPTION
**Pull request by mulkieran**

### CODE REVIEW

**Comment by mulkieran** [comment]
@drckeefe @bmr-cymru Please take a look and let me know if you feel satisfied w/ the command-line arguments for the get-metadata call.

**Review by drckeefe — APPROVED** [review]

File: src/stratis_cli/_actions/_debug.py [hunk 1]
```diff
@@ -153,6 +153,53 @@ def get_object_path(namespace):
...
+                "current": not namespace.written,
+            },
+        )
+
+        if return_code != StratisdErrors.OK:  # pragma: no cover
+            raise StratisCliEngineError(return_code, message)
+
+        json_format_data = [
+            json.loads(chunk) for chunk in metadata.split("\n") if chunk != ""
+        ]
```
Comment (jbaublitz) [inline src/stratis_cli/_actions/_debug.py:194]: This will have to change if the stratisd side changes.

**Review by jbaublitz — CHANGES_REQUESTED** [review]
There might be one change required due to the stratisd side.

**Review by jbaublitz — APPROVED** [review]

**Pull request merged**